### DISCUSSION TITLE
duplicate symbol 'nlohmann::detail::combine(unsigned long, unsigned long)'

### DISCUSSION THREAD

**Discussion by bettar** [nlohmann/json#2302] [Q&A] [answered]
If I use `#include <nlohmann/json.hpp>` in more than one file of the same application I get the error in the title. What am I doing wrong ?

**Comment by nlohmann** [comment, marked as answer]
Can you please try the latest develop version? This issue was fixed lately.

**Reply by bettar** [reply]
Yes, after updating from git commit a10d486e to the latest a048b72f it's now possible to include the header file more than once. Thank you.

**Reply by nlohmann** [reply]
#2301 will finally add a regression test for having multiple includes.
